packages/data-context/test/unit/sources/ProjectDataSource.spec.ts

```diff
@@ -563,16 +565,11 @@ describe('_makeSpecWatcher', () => {
 
     specWatcher.on('add', (filePath) => allFiles.add(filePath))
     specWatcher.on('change', (filePath) => allFiles.add(filePath))
+    specWatcher.on('unlink', (filePath) => allFiles.delete(filePath))
```

Comment:
@tgriesser I didn't think we'd be seeing this test so soon again :P I had to modify this test to pass after making so much stuff `async`, and this line was missing. I think the test is still testing what it's supposed to, can you verify?